bpftool.yaml

```diff
@@ -0,0 +1,68 @@
+package:
+  name: bpftool
+  version: 7.2.0
+  epoch: 0
+  description: Linux kernel bpf manipulation tool
+  copyright:
+    - license: GPL-2.0-only OR BSD-2-Clause
+  dependencies:
+    runtime:
+      - libLLVM-16
```

Comment:
I may be wrong, but off the top of my head this doesn't seem necessary?

Reply:
Hmmm, otherwise binary complains with `bpftool: error while loading shared libraries: libLLVM-16.so: cannot open shared object file: No such file or directory`